app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
In the code you've submitted, only the "last updated" date is aligned in a separate column. The "created" date is not.

![lists-date](https://cloud.githubusercontent.com/assets/221850/15450000/f61ea6ea-1f8e-11e6-84da-c277f2ac9a00.png)

Your initial code was showing like above, except without a comma between the author and creation date. Then I pointed out it looks strange with only a space separating the author and the date, so you added a comma (screenshot above).

If we agree to keep this layout, then what I was saying is that you should change `__('created by {listAuthor}', true)` into `__('created by {listAuthor}, {date}', true)` for the reason I explained in my previous comment.

If what you meant to say (but that I failed to understand) is that you'd rather have the created date aligned in its own column, just like the "last updated" date, then I'm okay with it too. It gives a bit less space for the title, but it's not too big of a deal. In this case you can remove the comma, put the created date in its own `<td>` and add a title `created` on the column.

I hope we've managed to come to a common understanding ^^;